### PR TITLE
Feature/DTGB-258: Updated iconfont

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I have updated the documentation accordingly.
- [x] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/components/21-atoms/icons/icon/README.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
 # Gent icon font
 
+## General
+
 For more information on the use of the Gent icon font see the notes in the
 `Base -> Fonts section`
+
+## a11y
```
Comment (delrueba) [inline styleguide/components/21-atoms/icons/icon/README.md]: Change to Web accessibility.
Not all readers are familiar with 'a11y'.

**Review by delrueba — CHANGES_REQUESTED** [review]

**Review by Jeroen005 — APPROVED** [review]

File: styleguide/components/21-atoms/icons/icon/README.md [hunk 2]
```diff
@@ -1,4 +1,12 @@
...
 
+## General
+
 For more information on the use of the Gent icon font see the notes in the
 `Base -> Fonts section`
+
+## Web accessibility
+
+* Icons are not used as the only means to convey the information. They can only
+  be used when their context also defines their meaning.
```
Comment (delrueba) [inline styleguide/components/21-atoms/icons/icon/README.md]: I believe this last sentence is a bit redundant.
I propose:
* Icons are not used as the only means to convey the information, i.e. icons are decorative.
* The icon must have the aria-hidden=true attribute.

File: styleguide/components/21-atoms/icons/icon/README.md (hunk 1 above)
Comment (meirege) [reply styleguide/components/21-atoms/icons/icon/README.md]: Done.

File: styleguide/components/51-templates/icons-overview/icons-overview.config.json [hunk 3]
```diff
@@ -37,7 +37,10 @@
       "star-full": "star-full",
       "star-half": "star-half",
       "favicon": "favicon",
-      "logo": "logo"
+      "logo-nl": "logo-nl",
+      "logo-en": "logo-en",
+      "logo-fr": "logo-fr",
+      "logo-sp": "logo-sp"
```
Comment (Jeroen005) [inline styleguide/components/51-templates/icons-overview/icons-overview.config.json]: sp? 🤔

**Review by Jeroen005 — CHANGES_REQUESTED** [review]

File: styleguide/components/51-templates/icons-overview/icons-overview.config.json (hunk 3 above)
Comment (delrueba) [reply styleguide/components/51-templates/icons-overview/icons-overview.config.json]: Nice catch.
Let's use ISO format: es
https://www.w3schools.com/tags/ref_language_codes.asp

Comment (meirege) [reply styleguide/components/51-templates/icons-overview/icons-overview.config.json]: Fixed.

**Review by delrueba — APPROVED** [review]

**Review by Jeroen005 — APPROVED** [review]

**Pull request merged**